shared/js/dataDriver.js

```diff
@@ -284,6 +301,7 @@ loop.DataDriver = function() {
       }));
       this._sse.addEventListener("open", this);
       this._sse.addEventListener("put", this);
+      this._sse.addEventListener("error", this);
```

Comment:
What code is fetching one line of chat? on `_connect` it should be updating the time skew and gets the last 24 hours of chat.